### PR TITLE
Adding template page for detailed statistics

### PR DESCRIPTION
**Pull request by dambem**
## Relevant issue
Want to add some information that's more detailed (e.g long term statistics)
## Checklist
- [x ] Tests all pass

### CODE REVIEW

File: ClimateMonitorV2/routes/index.js [hunk 1]
```diff
@@ -18,6 +18,9 @@ router.get('/', function (req, res) {
 router.get('/about', function (req, res) {
     res.render('about', { title: 'About SheffSenseV2' });
 });
+router.get('/detailedstats', function (req, res) {
+    res.render('detailedstats', { title: 'About SheffSenseV2' });
```
Comment (JonahIBM) [inline ClimateMonitorV2/routes/index.js:22]: { title: 'SheffSenseV2 statistics' } maybe?

**Review by M-Davies — APPROVED** [review]
LGTM

File: ClimateMonitorV2/routes/index.js (hunk 1 above)
Comment (JonahLuckett) [inline ClimateMonitorV2/routes/index.js]: Should this title be something like { title: 'SheffSenseV2 Statistics' }

Comment (dambem) [reply ClimateMonitorV2/routes/index.js]: Updated with correct title

**Pull request merged**